### DISCUSSION TITLE
【2024.2.1】community meeting notes

### DISCUSSION THREAD

**Discussion by cubxxw** [openimsdk/open-im-server#1856] [meeting]
**Meeting replay:** [Bilibili]()

**Conference link：**
> **Note**: Every two weeks at 9:00 p.m
https://meeting.tencent.com/dm/y6D9zgQbcXmF 📱#Tencent Conference：333-9699-4866

**Outline of the meeting:**
- [User case studies](https://github.com/OpenIMSDK/community/blob/main/ADOPTERS.md) and [Register the use of OpenIM](https://github.com/OpenIMSDK/Open-IM-Server/issues/379)
- This section describes the OpenIMSDK project
- What are we doing now
- Overall project planning and [roadmap for OpenIM](https://github.com/orgs/OpenIMSDK/projects/4/views/2)
- We also have Slack channels for you to communicate and discuss. To join, visit https://slack.com/ and join our [👀 Open-IM-Server slack](https://join.slack.com/t/openimsdk/shared_invite/zt-1tmoj26uf-_FDy3dowVHBiGvLk9e5Xkg) team channel.
- [Contribution guide](https://github.com/OpenIMSDK/Open-IM-Server/blob/main/CONTRIBUTING.md), describes the entire contribution step
- Summary of the meeting, [summary of the document](https://docs.google.com/document/d/1nx8MDpuG74NASx081JcCpxPgDITNTpIIos0DS6Vr9GU/edit?usp=sharing)

**If you have a topic to discuss, please comment below.**

**Comment by cubxxw** [comment]
We are excited to announce that we have prepared a special New Year gift package for all our valued contributors. To ensure that we can deliver the gifts to the right place, we kindly ask you to register your mailing address using the following form: [Contributor Registration Form](https://noteforms.com/forms/openim-developer-contributor-registration-lkeft4).

Thank you for your continued support and dedication to our project. We truly appreciate all that you do.

**Comment by cubxxw** [comment]
Some of the latest technical documentation for OpenIM has appeared ！

Mainly include the following：

+ OpenIM architecture Introduction document
+ Overall architecture diagram of OpenIM
+ Overall message flow diagram for OpenIM

**Reply by cubxxw** [reply]
@cubxxw

**Comment by cubxxw** [comment]
At present, OpenIM is working on some small projects, and we are appealing to contributors to participate or contribute to our cause. 

 News! In 2 weeks, we're launching our First training camp. The course begins with a comprehensive introduction to OpenIM. Topics include OpenIM's functionality, architecture, SDKs, deployment options, and what sets us apart from other IM solutions. The session also conclude with an engaging Q&A where you can dive deeper into OpenIM.

<img src="https://media0.giphy.com/media/hZj44bR9FVI3K/giphy.gif"/>

**Reply by cubxxw** [reply]
@AndrewZuo01

**Comment by cubxxw** [comment]
I would appreciate your assistance in gathering and discussing the main directions and roadmap for the new year in OpenIM. Your insights and contributions to this discussion would be highly valuable. 

> [!TIP]
> If you have suggestions or guidance for the future path of OpenIM, please comment below this topic

**Reply by cubxxw** [reply]
I propose that in the future, OpenIM could potentially integrate some advanced AI capabilities.

**Comment by cubxxw** [comment]
Dear contributors,

We are excited to announce that we not only offer courses for our overseas contributors, but also provide a growth program for our domestic contributors. The program, named "Different Contributor," will feature a multitude of asynchronous issue releases for developers and contributors to participate in learning. We will provide courses, guidance, and a communication group for interaction. Join us in this exciting opportunity to grow and learn together!

Best regards,
@openim

**Comment by skiffer-git** [comment]
OpenIM, in conjunction with LangChain, offers an intelligent customer service system based on a local knowledge base, supporting local LLM and GPT, embedding iframe; the code is open-source and ready to use out of the box.

**Reply by gsoec** [reply]
where could we find the code please?
